test/tstRay.cpp

```diff
@@ -250,9 +250,14 @@ BOOST_AUTO_TEST_CASE(ray_box_distance)
 {
   using ArborX::Box;
   using ArborX::Experimental::Ray;
-
+// use const instead of constexpr because MSVC shows error(error:expression
+// must have a constant value)
+#ifdef _MSVC_VER
+  const Box unit_box{{0, 0, 0}, {1, 1, 1}};
```

Comment:
An alternative would be to define a macro
```c++
#ifdef _MSVC_VER
#define ARBORX_MSVC_CONSTEXPR const
#else
#define ARBORX_MSVC_CONSTEXPR constexpr
#endif
```
and then do things like
```
  ARBORX_MSVC_CONSTEXPR Box unit_box{{0, 0, 0}, {1, 1, 1}};
```

Reply:
We looking at this more but it looks like we may not need to change this constexpr after all. We may only need to change the `inf`.